### ISSUE TITLE
Invalid error message when did is too long

### ISSUE THREAD

**Issue by Warchant** [soramitsu/did-resolver#85]
There should be some constraint on DID length, because it is stored as "key" in iroha, which is limited to 64 symbols.

Currently error message is the following:
```
{"timestamp":"2018-10-16T07:37:15.622+0000","status":500,"error":"Internal Server Error","message":"java.util.concurrent.ExecutionException: jp.co.soramitsu.sora.didresolver.exceptions.IrohaTransactionCommitmentException: Cannot commit transaction with hash: 4cb778bcacded973f520c86e2b97886fefb930c838ce17bdf7a5eedb75d5ca07. Last status: STATELESS_VALIDATION_FAILED","path":"/did"}
```

Should be something like:
```
did is too long
```

**Comment by domoininnopolis** [comment]
https://github.com/soramitsu/did-resolver/pull/87

**Comment by domoininnopolis** [comment]
https://github.com/soramitsu/did-resolver/pull/87

**Comment by Warchant** [comment]
@domoininnopolis you can use https://help.github.com/articles/closing-issues-using-keywords/ to close issues.

This one is:

"Closes #85" in your PR
